### ISSUE TITLE
Permit Exceptions

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PaDS#13]
@adistasio @jwoosnam 

We have some instances which need to be individually given permits to read.  I think you said you have this ability in PaDS.  By design, any article with an article ID ending in page 0000A is actually a TOC instance and should be freely open for viewing.  

This was intended for split books, but looking at the data set, there are 4 articles or so, all editorial board lists, which were assigned page 0 (as above).  Those are fine to show in all case as well, if you want to make it a simple universal rule instead of list the cases.  Right now, there are 167 instances listed as Page 0 (one for each volume of GW, SE, and many of the ZBKs).

This is how PEP-Web currently works.  And this will solve the problem with book browse for split books.

**Comment by jwoosnam** [comment]
Yes the new system can give access to individual documents, but for this I think the most flexible is to have a Universal Permits exception driven by a regular expression match against the full document ID.  For now it will just allow docs ending 0000A, but could be extended via the regular expression pattern stored as a database parameter.
I hope to implement this later today.
